useful-scripts/run.js

```diff
@@ -34,7 +34,7 @@ const projects = {
       projectRoot,
       'src/GovUk.Education.ExploreEducationStatistics.Admin',
     ),
-    command: 'dotnet build && dotnet run',
+    command: 'dotnet clean && dotnet run',
```

Comment:
Do the cleans essentially stop the AspectInjector errors we sometimes see when running things simultaneously?

Reply:
Unfortunately not - the shared build file issue is still an issue... he says, going off to check... yup, it still is unfortunately:

```
[content] AspectInjector|2.7.3 : error AI_ERR0: Processing failure: System.IO.IOException: The process cannot access the file '/home/dwatson/development/projects/dfe/explore-education-statistics/src/GovUk.Education.ExploreEducationStatistics.Content.Services/obj/Debug/net6.0/GovUk.Education.ExploreEducationStatistics.Content.Services.pdb' because it is being used by another process. [/home/dwatson/development/projects/dfe/explore-education-statistics/src/GovUk.Education.ExploreEducationStatistics.Content.Services/GovUk.Education.ExploreEducationStatistics.Content.Services.csproj]
[content] AspectInjector : error AI_FAIL: Aspect Injector processing has failed. See other errors. [/home/dwatson/development/projects/dfe/explore-education-statistics/src/GovUk.Education.ExploreEducationStatistics.Content.Services/GovUk.Education.ExploreEducationStatistics.Content.Services.csproj]
```

What this clean basically does is ensures that there's no compiled code pre-existing before the build that is triggered by `dotnet run`.

It seems that for every potential command that could cause a compile, the AI compilation step occurs whether a class needs recompilation or not, and it is applied over the already-decorated code.  So previously, doing a `dotnet build && dotnet run` caused 2 AI compilation steps to occur, thus resulting in 2 loads of aspect code wrapping places in the code that were annotated with [BlobCache] (and so 2 gets, 2 sets).  Without cleans, the next set of `dotnet build && dotnet run` would then cause 4 gets, 4 sets.  And so on.

This approach above is still not ideal unfortunately - trying to do multiple `dotnet clean && dotnet run` simultaneously with no staggering will cause issues as different projects attempt to clean down shared projects as other projects are in the process of using them.  But it is necessary with the updated AspectInjector library to do this as it will no longer silently wrap Aspect code in more Aspect code - it'll now blow up!  So you have to clean it down first.